guide/event-driven-and-pooling-consumer-service/tests/event-driven-and-pooling-consumer_test.bal

```diff
@@ -0,0 +1,173 @@
+// Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/io;
+import ballerina/mb;
+import ballerina/runtime;
+import ballerina/task;
+import ballerina/test;
+
+string requestmessage1;
+string requestmessage2;
+string requestmessage3;
+string requestmessage4;
+
+documentation { Define the message queue endpoint for sending geo activities captured from the sensor. }
+endpoint mb:SimpleQueueSender queuesensorGeoMessage {
```

Comment:
endpoint name should be camelcase. like `queueSensorGeoMessage`

Reply:
Change the queuesensorGeoMessage to camelcase.